### PR TITLE
Gang Mode Update 4

### PR DESCRIPTION
**Pull request by Ikarrus**
**Major Changes**
- Increased win condition to require 66% control
- Added a switchblade. A relatively cheap and decently robust melee weapon for gangsters.
- Limited gang spraycan use to 15
- Promotions now start off cheap at 20 influence but get more expensive the more people you promote. As a consequence, you can only promote up to 3 lieutenants.

**Minor Changes**
- Increased pen cooldown to (15*Gang size)+45 seconds
- Tagging a territory now tells you what territory you actually tagged!
- Added an option to silence the verbose territorial updates you get from gangtools
- Capped how much influence you can get every income at 50
- Updated Centcom report on gangs
- Spraycans always show their inhands now, instead of only when uncapped
- Antag panel now shows how much influence each gang has
- Cleaned up gang code in the traitor panel
- Added logging for gangtool purchases
- Promotion cost now counts how many times you bought that item instead of how many bosses you have, so you can't stockpile on cheap gangtools before promoting anyone
- Fixed non-gangers getting gang messages when trying to spray on an invalid territory with a gang spraycan

### CODE REVIEW

**Comment by Lumipharon** [comment]
Fuck yeah, switchblades were fun as shit on NT.

Could they be constructable also? If nothing else but for a bit of anti meta. NT had them need metal and plasteel, so they weren't super easy to make (although eva tended to get raided a bit).

**Comment by Alek2ander** [comment]
Hacked autolathe?

**Comment by tedward1337** [comment]
Fuck yeah, switchblades

**Comment by ike709** [comment]
make switchblades constructable. it brings us one step closer to NT's glorious gun cargo.

:+1:

**Comment by Fox-McCloud** [comment]
I don't think a concealable weapon that can do, literally, 20 damage per stab-----would be a good thing to be constructable by crewmembers.

The best weapon available to crew is the fire-axe---which does 24 and requires it to be *wielded*.

**Comment by Cookiesndip** [comment]
> Increased pen cooldown to (15*Gang size)+45 seconds

What was it previously, for scale? I've found that very little of the station actually winds up in a gang even with zero implanting going on based on rounds yesterday. Reducing the availability of implants is a whole new can of worms and would be entailing entirely stripping rev, so let's not even consider that. Optimally, the two gangs together should have the numbers and the strength to fight on par with security in the midgame (by then everything is tagged, borders drawn, etc.) if they become an obstruction/forceimplant folks.

Constraining the paint to 15/per purchase without lowering the price in proportion, will excaberate this further, as paint will accelerate victory AND provide additional/protect your income, whereas, say, a pistol is almost completely useless at such a high price, being able to kill/protect you from 2~ people, when a slip/stunbaton and saw or welder fluid + lighter would be just as effective, or one of the billion other combat options.

Optimally, taggers should be the most expensive items, by far, whereas consumables (syndicate pistols with limited ammo and switchblades esp.) should be markedly cheaper. There isn't much you can do with a pistol with no extra ammo (and a slip + saw is just as effective), and, unless the switchblade is markedly better than normal melee units, you're better off not buying it and using a station-available item instead. Perhaps, if switchblades do not have an onhand sprite and have saw-level damage, that might be enough to justify purchase and opportunity cost.

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -158,4 +158,40 @@ obj/item/weapon/wirerod/attackby(var/obj/item/I, mob/user as mob, params)
 
 
 
-
+/obj/item/weapon/switchblade
+	name = "switchblade"
+	icon_state = "switchblade"
+	desc = "A sharp, concealable, spring-loaded knife."
+	flags = CONDUCT
+	force = 20
```
Comment (phil235) [inline code/game/objects/items/weapons/weaponry.dm]: You weren't joking when you said these were robust!

Comment (Miauw62) [inline code/game/objects/items/weapons/weaponry.dm]: 20 is really robust. make it like 16 or something, it'd still be pretty robust.

Comment (Ikarrus) [inline code/game/objects/items/weapons/weaponry.dm]: 16 is just barely stronger than a welding tool, which at that point makes the item utterly useless to spend influence on.

Comment (Miauw62) [inline code/game/objects/items/weapons/weaponry.dm]: as somebody else noted, 20 is only 4 less than a fire axe and those have to be wielded.

Comment (Ikarrus) [inline code/game/objects/items/weapons/weaponry.dm]: And I'm fine with that! 20 is what I would consider to be the minimally acceptable force for the weapon to have any value at all.

Keep in mind that in order to get one you have to spend *valuable* influence that isn't just to buy toys, but also to keep your gang going with spraycans.

**Comment by Fox-McCloud** [comment]
For the record, I"m not suggesting that gang version be nerfed, just that the gang version of 20 brute NOT be constructable/obtainable by crew and that something that you can conceal in your pocket that does that much damage (and is for non-antags) is OP.

**Pull request merged**